app/brokers/(3-techincal-docs)/javascript-sdk/functions/page.mdx

```diff
@@ -757,6 +766,7 @@ const status = await swapSDK.getStatusV2({ id: receipt.hash });
   sourceTokenAddress: '0x1c7d4b196cb0c7b01d743fbc6116a902379c7238'
 }
 ```
+</details>
 
 #### Bitcoin vault swap (Boosted)
```

Comment:
This is also an example (as same for Solana). Maybe we could have them as separate collapsible sections titled accordingly e.g. "Example: ethereum vault swap" etc.